man/man7/zpoolprops.7

```diff
@@ -40,6 +40,15 @@ Each pool has several properties associated with it.
 Some properties are read-only statistics while others are configurable and
 change the behavior of the pool.
 .Pp
+When displaying properties for sizes in human-readable mode, to conserve
+display space, values will be printed in powers of 1024, with a single letter
+indicating the unit prefix followed immediately after the value. For example
+a human-readable output of the
+.Sy allocated
+property
+.Ar 16.0G
```

Comment:
IEC is very clear that G means `1000^3`. If we adopt the IEC convention, we must write `Gi`. Anything else would be non-standard.